README.md

```diff
@@ -1,30 +1,85 @@
 # NUpbr
+
 This repository holds code to generate a configurable, Physically Based Rendered (PBR) football field.
 
 ![Field Example](./docs/outputs/goals_example.gif)
 
 # Requirements
-+ [Blender](https://www.blender.org/download/) (2.79)
+
+-   [Blender](https://www.blender.org/download/) (2.79)
 
 # Usage
```

Comment:
The way that github renders this is weird. I would just remove it, or change the document structure more